sdk/core/Azure.Core/src/Pipeline/ServicePointHelpers.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using System.Net.Http;
+
+namespace Azure.Core.Pipeline
+{
+    internal static class ServicePointHelpers
+    {
+        private const int RuntimeDefaultConnectionLimit = 2;
+        private const int IncreasedConnectionLimit = 50;
```

Comment:
Would there be any value in allowing this to be overridden by an environment variable?

Reply:
Or should we let it override through `Azure.Core.ClientOptions`?

Reply:
I would start without it and see if anyone cares.

Reply:
To add some context, there are already global ways to set this value via ServicePointManager or app config.

And I'm not sure we want to add it to options because it's hard to make it work with custom-provided transports.